Code/Tools/AssetProcessor/native/tests/PathDependencyManagerTests.cpp

```diff
@@ -68,21 +93,40 @@ namespace UnitTests
         // ":memory:" databases are one-instance-only, and even if another connection is opened to ":memory:" it would
         // not share with others created using ":memory:" and get a unique database instead.
         m_databaseLocation = tempPath.absoluteFilePath("test_database.sqlite").toUtf8().constData();
-
-        ON_CALL(m_databaseLocationListener, GetAssetDatabaseLocation(_))
-            .WillByDefault(
-                DoAll( // set the 0th argument ref (string) to the database location and return true.
-                    SetArgReferee<0>(m_databaseLocation),
-                    Return(true)));
+        m_databaseLocationListener.m_databaseLocation = m_databaseLocation;
 
         m_stateData = AZStd::shared_ptr<AssetProcessor::AssetDatabaseConnection>(new AssetProcessor::AssetDatabaseConnection());
         m_stateData->OpenDatabase();
 
         m_platformConfig = AZStd::make_unique<AssetProcessor::PlatformConfiguration>();
+
+        AZ::AllocatorInstance<AZ::PoolAllocator>::Create();
+        AZ::AllocatorInstance<AZ::ThreadPoolAllocator>::Create();
+
+        AZ::JobManagerDesc jobDesc;
+        AZ::JobManagerThreadDesc threadDesc;
+        for (int i = 0; i < 16; ++i)
```

Comment:
what's the significance of 16 here? Can you comment it, or better yet make it a const?

Reply:
I've replaced both of these with just loading the JobManagerComponent, which will use 1 thread per core.